hudi-spark/src/main/scala/org/apache/hudi/HoodieSparkUtils.scala

```diff
@@ -27,9 +27,9 @@ import org.apache.spark.sql.types.{StringType, StructField, StructType}
 import scala.collection.JavaConverters._
 
 
-object HudiSparkUtils {
+object HoodieSparkUtils {
```

Comment:
@vinothchandar Looks like some unrelated change was added during the rebase. Maybe this is related to the issue.

Reply:
nvm, this was intended based on previous comments to keep the consistency with others.

Reply:
this was something that i should have caught in the earlier pr . I renamed it here, thinking we will be landing this quickly. oh well. lets just get the test to pass